app.js

```diff
@@ -17,6 +17,16 @@ function reveal() {
   }
 }
 
+$('li.navigation:eq(0)').on('click', function(){
+  $('.content').show();
+  $('html, body').animate({scrollTop: 0})
+})
+
+$('li.navigation:eq(1)').on('click', function(){
+  $('.content').hide();
+  $('.main').appendTo('body');
```

Comment:
You should not be appending `.main` to the DOM repeatedly. Since it is already there, try showing/hiding it instead. Also, you should try to make an event handler for all the nav `li` elements that show the relevant content by making use of a data-attribute like in the lab code. This would make your nav more flexible so that if you ever add/remove tabs, you would not need to change your handler.